linode/manager
# feat: [M3-8834] - Display new Accelerated plans in Plans Panel

## Description 📝
Adds Accelerated plans to Plans Panel

## Changes  🔄
- Introducing a feature flag, acceleratedPlans + hide everything behind feature flag (and account capability)
- Update Plans Panel to match figma mocks (see internal ticket) for Accelerated plans
- Update mock regions data to include accelerated capability for some regions

## Target release date 🗓️
12/10

## Preview 📷
<img src="https://github.com/user-attachments/assets/67621b51-69e1-44cc-a916-38ef91233d44" width="600" />

## How to test 🧪
Either use the MSW, or make sure you have the correct customer tag (reach out for it)
- [ ] Confirm UI matches figma mockup for Linode plans (see internal ticket)
- [ ] Confirm Accelerated plans are hidden when feature flag is false (inside `useFlags`, test when different variations of `{ linodePlans: boolean; lkePlans: boolean }` are returned)
- [ ] ```yarn test PlansPanel/utils.test.ts```
- [ ] NOTE: there are still a few open questions (doc links) - created M3-8931 as a sanity check for everything. will update this branch as I get the answers tho)

## As an Author, I have considered 🤔

- 👀 Doing a self review
- ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- 🤏 Splitting feature into small PRs
- ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- 🧪 Providing/improving test coverage
- 🔐 Removing all sensitive information from the code and PR description
- 🚩 Using a feature flag to protect the release
- 👣 Providing comprehensive reproduction steps
- 📑 Providing or updating our documentation
- 🕛 Scheduling a pair reviewing session
- 📱 Providing mobile support
- ♿  Providing accessibility support

<br/>


## As an Author, before moving this PR from Draft to Open, I confirmed ✅

    - databaseResize having some flake
